app/services/listing_service/store/shapes.rb

```diff
@@ -0,0 +1,140 @@
+module ListingService::Store::Shapes
```

Comment:
We've seemed to establish a convention of using singular form to name stores, Shape vs Shapes, Store::PaypalPayment, Store::Transaction, etc.